workflow/rules/tmb.smk

```diff
@@ -20,13 +20,13 @@ rule tmb:
         filter_nr_observations=config.get("tmb", {}).get("filter_nr_observations", 1),
         DP_limit=config.get("tmb", {}).get("DP_limit", 200),
         VD_limit=config.get("tmb", {}).get("VD_limit", 10),
-        AF_lower_limit=config.get("tmb", {}).get("AF_lower_limit", 0.02),
+        AF_lower_limit=config.get("tmb", {}).get("AF_lower_limit", 0.05),
         AF_upper_limit=config.get("tmb", {}).get("AF_upper_limit", 0.45),
         GnomAD_limit=config.get("tmb", {}).get("GnomAD_limit", 0.0001),
         db1000G_limit=config.get("tmb", {}).get("db1000G_limit", 0.0001),
         Background_sd_limit=config.get("tmb", {}).get("Background_sd_limit", 5),
-        nsSNV_TMB_correction=config.get("tmb", {}).get("nsSNV_TMB_correction", 0.625),
-        nsSNV_sSNV_TMB_correction=config.get("tmb", {}).get("nsSNV_sSNV_TMB_correction", 0.625),
+        nsSNV_TMB_correction=config.get("tmb", {}).get("nsSNV_TMB_correction", 0.78),
```

Comment:
For me, this looks okay but I don't really get why changing the default is required here. I mean you could easily change it in your `config.yaml`. Maybe we should talk about this. Another thing I came to think of is that we agreed on lower case for everything possible which includes all the parameters as well :wink:

Reply:
For some reason I have put it in here instead of the config.

Reply:
I have now added it to pipeline and corrected the lower case.

Reply:
looks good!